docs/install.rst

```diff
@@ -4,7 +4,7 @@ Install Tendermint
 From Binary
 -----------
 
-To download pre-built binaries, see the `Download page <https://tendermint.com/download>`__.
+To download pre-built binaries, see the `Download page <https://tendermint.com/downloads>`__.
```

Comment:
the downloads page just points to github releases ... should we maybe point to the GH releases instead?

Reply:
we might switch to something else.. lets keep it for a while